examples/frigate/frigate.py

```diff
@@ -23,15 +25,43 @@ def frigate_events(topic, data, srv=None):
     })
     return r
 
-def frigate_events_filter(topic, message, section, srv=None):
+
+def frigate_events_filter(topic, message, section, srv: Service=None) -> bool:
+    """
+    mqttwarn filter function to only use Frigate events of type `new`.
+    Additionally, skip, for example, false positives.
+
+    :return: True if message should be filtered, i.e. notification should be skipped.
+    """
+
+    # Decode message.
     try:
         message = json.loads(message)
     except:
-        message = None
-    if message and message.get('type', None) != 'end' and 'after' in message:
-        a = message['after']
+        message = dict()
+    message_type = message.get("type", None)
+    srv.logging.info(f"Received Frigate event message with type={message_type}")
+
+    # Determine if message should be used, or not.
+    use_message = message_type == "new" and "after" in message
+
+    # Look at more details.
```

Comment:
These changes are likely much better for an example, rather than my arcane list comprehensions :)

Looks good

Reply:
Thanks. As outlined over at https://github.com/jpmens/mqttwarn/issues/632#issuecomment-1506010580, 58f716d58 attempts to bring your most recent snippet into this shape. I hope it will be for the better.